scripts/ci/e2e_benchmark.py

```diff
@@ -36,10 +36,10 @@ def run(self, benchmark_name, host, num_requests, warmup_requests=50):
             response = requests.get(url)
             end_time = time.time()
             if response.status_code != 200:
-                if benchmark_name == 'match':
-                    code = response.json()['code']
-                    if code == 'NoSegment' or code == 'NoMatch':
-                        continue
+                code = response.json()['code']
```

Comment:
Some requests in benchmark started returning error code which had no before it and it is not due to this change :) I have no idea why we started having it 🤔 
I have 2 theories: 
- berlin-latest.osm.pbf was updated (we don't store it in the repo, but just cache it in GH runner's cache)
- something was updated on runner

But in general it is expected that some of requests return such errors, so I think it is okay to have it updated.